realtime/server.js

```diff
@@ -133,7 +144,7 @@ socket.on('connection', function (client) {
             else {
                 messageid = body;
 
-                messages_typing[messageid] = form;
+                messages_typing[messageid] = form; //set the info for the message that user is going to write
```

Comment:
Not sure why this is a message the user "is going to write", as the message portion transmitted has already been typed out.